src/test/java/fr/inria/coming/spoon/repairability/RepairabilityTest.java

```diff
@@ -24,6 +24,11 @@ public void testRepairabilityInterface() throws Exception {
         TestUtills.runRepairability("JMutRepair", "/repairability_test_files/one/");
     }
 
+    @Test
+    public void testDiffResults() throws Exception {
+        TestUtills.runRepairabilityPrint("ALL", "repogit4testv0");
```

Comment:
thanks for the progress.

add an assertion checking that a diff file has been created on diff and that it contains the correct diff.

Reply:
` 

@Test
    public void testDiffResults() throws Exception {
        FinalResult result = TestUtills.runRepairabilityPrint("ALL", "repogit4testv0");
        Map<IRevision, RevisionResult> revisionsMap = result.getAllResults();
        assertEquals(13, revisionsMap.keySet().size());

        for (Map.Entry<IRevision, RevisionResult> entry : revisionsMap.entrySet()) {
            RevisionResult rr = entry.getValue();
            PatternInstancesFromRevision instances = (PatternInstancesFromRevision) rr.getResultFromClass(RepairabilityAnalyzer.class);
            List<DiffRow> row_list = instances.getRow_list();
        }


        }

`


This is the test case now, I don't understand how to check if the result is indeed correct (other than looking at the produced JSON file)? I mean I don't know what to compare row_list to?

Reply:
> other than looking at the produced JSON file

That's OK to look at the content of the JSON file.

Reply:
so is the test case complete?